package/Dockerfile

```diff
@@ -1,7 +1,20 @@
 ARG RANCHER_BASE_TAG=base
-ARG RANCHER_REPO=rancher
+ARG RANCHER_BASE_REPO=rancher
+
+FROM ghcr.io/oracle/oraclelinux:8-slim as build
+ARG ARCH
+ENV ARCH=$ARCH
+
+COPY . /go/src/github.com/rancher/rancher/
+
+RUN microdnf update -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs && \
+    microdnf install -y git && \
+    microdnf install -y golang-1.20.10-1.module+el8.9.0+90068+08775a1f.x86_64 && \
+    cd /go/src/github.com/rancher/rancher/scripts && \
```

Comment:
Should this be added:

    microdnf clean all && \
    rm -rf /var/cache/yum /var/lib/rpm/*

Reply:
actually it is not necessary since there is a FROM later in the file

Reply:
I will remove that before I merge